app/hooks/getNftContractAddress.ts

```diff
@@ -0,0 +1,10 @@
+import { useSmartAccountClient } from "@account-kit/react";
+import { arbitrumSepolia } from "@account-kit/infra";
+import { chainNFTMintContractData } from "@/lib/chains";
+
+export const getNftContractAddress = () => {
```

Comment:
sry i didn't notice this earlier, but should this be a hook? (i.e. `useNftContractAddress`) since it's in the `hooks` dir.